client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -424,6 +428,8 @@ class DomainDetailsForm extends PureComponent {
 				return;
 			}
 
+			//this.props.updateContactDetailsCache( this.getAllFieldValues() );
+
```

Comment:
I think we can take this out. Originally I had it save into the Redux state on next/submit but we probably want to keep that data fresher with something close to real time if that doesn't hit the reducers too hard.